### ISSUE TITLE
Use common initialization for database adapters

### ISSUE THREAD

**Issue by daffl** [feathersjs/feathers-nedb#1]
To have all adapters work similar, we may want to change having to use `new` to instantiate a new service instance like:

```js
// Connect to the db, create and register a Feathers service.
app.use('todos', new nedbService('todos'));
```

to just a function call:

```js
// Connect to the db, create and register a Feathers service.
app.use('todos', nedb('todos'));
```

**Comment by daffl** [comment]
Oh nevermind you're using Uberproto right? I don't think the `new` is necessary at all. Mind if I update it in the docs?

**Comment by marshallswain** [comment]
Make it happen.  Sounds good.

**Comment by daffl** [comment]
Closed via e19c8ff10ae69179f61eb7ce84d2a2969e700ca3
